src/NUnitFramework/tests/Internal/Results/TestResultTests.cs

```diff
@@ -32,8 +32,8 @@ protected static void ReasonNodeExpectedValidation(TNode testNode, string reason
         {
             TNode? reason = testNode.SelectSingleNode("reason");
             Assert.That(reason, Is.Not.Null);
-            Assert.That(reason.SelectSingleNode("message"), Is.Not.Null);
-            Assert.That(reason.SelectSingleNode("message").Value, Is.EqualTo(reasonMessage));
+            Assert.That(reason!.SelectSingleNode("message"), Is.Not.Null);
+            Assert.That(reason.SelectSingleNode("message")!.Value, Is.EqualTo(reasonMessage));
```

Comment:
Null Suppression operators are not needed as the NUnit.Analyzer suppresses these because they have been checked for `Is.Not.Null`

Reply:
I believe I added them when the analyzers dozed off, they do that from time to time - on my VS......

Reply:
Hmm..  I still get the squibblies:  
![image](https://github.com/nunit/nunit/assets/203432/757d25e1-0978-4a8b-83b5-d4edbf4088b2)
The analyzer is loaded, I get the NUnit2045 on the line below. 

I see that this warning comes from Resharper.  That means that also Rider will give this warning.

Reply:
ReSharper/Rider don't know about NUnit. They raise their own. This is from pre-C# nullability. C# is now very good at checking this. We dropped ReSharper for my work as it no longer added value.
Can you disable nullability checking in ReSharper for a project with nullability enabled? Should it do so by default to prevent double reporting?
You could raise an issue with ReSharper if they allow suppressing their errors if a 3rd party analyzer knows more.